src/Filesystem/Wrappers/File.php

```diff
@@ -11,11 +11,32 @@ final class File implements FileContract
 
     public function __construct(string $path)
     {
-        $this->file = new SplFileObject($path, 'w');
+        touch($path);
```

Comment:
toto mě nepřipadá správné, aby inicializace už rovnou něco vytvořilo.

touch by se mělo vytvořit pokud použiji write.

content by mělo správně spadnout pokud soubor neexistuje.

Reply:
instance SplFileObject to vyžadovala, upravil jsem